client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -62,72 +73,172 @@ class Unlocked extends React.Component {
 		} );
-	};
+	}
 
 	render() {
 		const { translate } = this.props;
-		const { privateDomain, hasPrivacyProtection, manualTransferRequired, pendingTransfer } = getSelectedDomain( this.props );
-		let domainStateMessage = translate( 'Your domain is unlocked to prepare for transfer.' );
+		const { submitting, canceled, sentFromThisPage } = this.state;
+		const {
+			privateDomain,
+			hasPrivacyProtection,
+			manualTransferRequired,
+			pendingTransfer,
+			domainLockingAvailable,
+		} = getSelectedDomain( this.props );
 
-		if ( pendingTransfer ) {
-			domainStateMessage = translate( 'Your domain is pending transfer.' );
-		} else if ( hasPrivacyProtection && ! privateDomain ) {
-			domainStateMessage = translate( 'Your domain is unlocked and Privacy Protection has been disabled' +
+		const alwaysTransferrable = ! ( domainLockingAvailable || hasPrivacyProtection );
+		let domainStateMessage, statusMessage;
+
+		const canCancel = pendingTransfer || ! alwaysTransferrable;
+		const canSend = manualTransferRequired && sent;
+
+		// if pending, the only thing you can do is (try to) cancel the transfer
+		// otherwise, if it's not sent, that's the thing you need to do
+		// otherwise, you need to either give the auth code to your new registrar or wait for support.
+
+		let userActionRequired, domainStateMessage;
+
+		if ( pendingTransfer || canceled ) {
+			userActionRequired = null;
+		} else if ( ! sent ) {
+			userActionRequired = translate(
+				'Please click the button above to proceed with the transfer.' );
+		} else if ( manualTransferRequired ) {
+			userActionRequired = null;
+		} else {
+			userActionRequired = translate(
+				'You must provide your new registrar with your domain name ' +
+				'and transfer code to complete the transfer process.' );
+		}
+
+		let actionReport = null;
+		if ( sent ) {
+			actionReport = manualTransferRequired
+				? translate( 'The registry for your domain requires a ' +
+					'special process for transfers. Our Happiness Engineers ' +
+					'have been notified about your transfer request and will ' +
+					'be in touch shortly to help you complete the process.' )
+				: translate( 'We have sent the transfer authorization code ' +
+					'to the domain registrant\'s email address.' )
+		} else if ( canceled ) {
+			// If we've enabled privacy we'll be redirected
+			// to the locked page, so we only need the base message here.
+			actionReport = translate( 'We\'ve canceled your domain transfer.' );
+		}
+
+		let lockStateReport;
+
+		if ( domainLockingAvailable && hasPrivacyProtection && ! privateDomain ) {
+			lockStateReport = translate(
+				'Your domain is unlocked and Privacy Protection has been ' +
+				'disabled to prepare for transfer.' );
+		} else if ( domainLockingAvailable ) {
+			lockStateReport =translate( 'Your domain is unlocked to prepare for transfer.' );
+		} else if ( hasPrivacyProtection ) {
+			lockStateReport = translate(
+				'Privacy Protection for your domain has been disabled' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Your domain is unlocked and Privacy Protection has been disabled to prepare for transfer.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).